hydra/test_utils/launcher_common_tests.py

```diff
@@ -288,9 +289,10 @@ def sweep_2_jobs(
     with sweep:
         assert sweep.temp_dir is not None
         assert sweep.returns is not None
-        temp_dir = Path(sweep.temp_dir)
 
-        multirun_cfg_path = Path(temp_dir) / "multirun.yaml"
+        config_dir = Path(sweep.temp_dir)
```

Comment:
what is this fixing?

Reply:
nothing 😂 a legacy change which i need to revert.